src/features/onboarding/components/FirstOnboarding/TagGroup.tsx

```
@@ -0,0 +1 @@
+export const TagGroup = (props) => {};
```

ESLint

[error] 'props' is defined but never used. Allowed unused args must match /^_/u.

(@typescript-eslint/no-unused-vars)